Project-Starlord/Controllers/AccountController.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Project_Starlord.Data;
+using Project_Starlord.Models;
+
+namespace Project_Starlord.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        private readonly MyDbContext _context;
+
+        public AccountController(MyDbContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/UserModels
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<UserModel>>> GetUsers()
+        {
+            return await _context.Users.ToListAsync();
+        }
+
+        // GET: api/UserModels/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<UserModel>> GetUserModel(int id)
+        {
+            var userModel = await _context.Users.FindAsync(id);
+
+            if (userModel == null)
+            {
+                return NotFound();
+            }
+
+            return userModel;
+        }
+
+        // PUT: api/UserModels/5
+        // To protect from overposting attacks, please enable the specific properties you want to bind to, for
+        // more details see https://aka.ms/RazorPagesCRUD.
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutUserModel(int id, UserModel userModel)
+        {
+            if (id != userModel.Id)
+            {
+                return BadRequest();
+            }
+
+            _context.Entry(userModel).State = EntityState.Modified;
+
+            try
+            {
+                await _context.SaveChangesAsync();
+            }
+            catch (DbUpdateConcurrencyException)
+            {
+                if (!UserModelExists(id))
+                {
+                    return NotFound();
+                }
+                else
+                {
+                    throw;
+                }
+            }
+
+            return NoContent();
+        }
+
+        // POST: api/UserModels
+        // To protect from overposting attacks, please enable the specific properties you want to bind to, for
+        // more details see https://aka.ms/RazorPagesCRUD.
+        [HttpPost]
+        public async Task<ActionResult<UserModel>> PostUserModel(UserModel userModel)
+        {
+            _context.Users.Add(userModel);
+            await _context.SaveChangesAsync();
+
+            return CreatedAtAction("GetUserModel", new { id = userModel.Id }, userModel);
```

Comment:
Maybe you can use `nameof(GetUserModel)` instead of this hardcoded string, but it's not a requirement.